docs/03_server/05_access-control/03_password_authentication.md

```diff
@@ -530,51 +631,78 @@ Note the following:
     DETAILS.NAME = SALES_TRADERS
 
 #### Delete response
-    MESSAGE_TYPE = EVENT_DELETE_PROFILE_ACK
+    MESSAGE_TYPE = EVENT_ACK
 
 ### Insert User
 
 #### Insert request
     MESSAGE_TYPE = EVENT_INSERT_USER
-    USER_NAME = jwolf
+    USER_NAME = JohnDoe
     DETAILS.USER_NAME = JohnWolf
     DETAILS.FIRST_NAME = John
     DETAILS.LAST_NAME = Wolf
     DETAILS.EMAIL_ADDRESS = john.wolf@genesis.global
     DETAILS.STATUS = ENABLED
-    DETAILS.RIGHT[0].ID = 00000000000001RISP0
-    DETAILS.RIGHT[0].CODE = ORDEN
-    DETAILS.RIGHT[1].ID = 00000000000002RISP0
-    DETAILS.RIGHT[1].CODE = ORDAM
+    DETAILS.USER_PROFILES[0] = USER_ADMIN
 
 #### Insert response
-    MESSAGE_TYPE = EVENT_INSERT_USER_ACK
+    MESSAGE_TYPE = EVENT_ACK
 
 ### Amend user
 
+In the example below, the logged-in user (in the second line) is JohnDoe, who is modifying the profile of JohnWolf to give him the last name: Smith.
+
 #### Amend request
     MESSAGE_TYPE = EVENT_AMEND_USER
-    USER_NAME = jwolf
-    DETAILS.ID = 00000000000001USSP0
+    USER_NAME = JohnDoe
     DETAILS.USER_NAME = JohnWolf
     DETAILS.FIRST_NAME = John
-    DETAILS.LAST_NAME = Doe
+    DETAILS.LAST_NAME = Smith
     DETAILS.EMAIL_ADDRESS = john.wolf@genesis.global
     DETAILS.STATUS = ENABLED
-    DETAILS.RIGHT[0].ID = 00000000000001RISP0
-    DETAILS.RIGHT[0].CODE = ORDEN
-    DETAILS.RIGHT[1].ID = 00000000000002RISP0
-    DETAILS.RIGHT[1].CODE = ORDAM
+    DETAILS.USER_PROFILES[0] = USER_ADMIN
 
 #### Amend response
-    MESSAGE_TYPE = EVENT_AMEND_USER_ACK
+    MESSAGE_TYPE = EVENT_ACK
 
 ### Delete user
 
 #### Delete request
     MESSAGE_TYPE = EVENT_DELETE_USER
     USER_NAME = JohnDoe
-    DETAILS.USER_NAME = james
+    DETAILS.USER_NAME = JohnWolf
 
 ### Delete response
-    MESSAGE_TYPE = EVENT_DELETE_USER_ACK
+    MESSAGE_TYPE = EVENT_ACK
+
+## Full permissions list
+
+Most of the message workflows described in this page are permissioned based on a set of default rights which are provided in the form of a CSV file as part of the platform authentication distribution. Most permissions and their relationship to events have been explained in their own sections, and the whole table can be found below for reference:
```

Comment:
Adding the whole list of rights available for user management as well as special information about EXPIRE_PWD and CHANGE_PWD events.